cmd/gendeepcopy/deep_copy.go

```diff
@@ -92,7 +100,14 @@ func main() {
 			generator.OverwritePackage(vals[0], vals[1])
 		}
 	}
-	for _, knownType := range api.Scheme.KnownTypes(version) {
+	//handle the special cases of groupVersion = "api/" and "experimental/"
+	if version == "" {
```

Comment:
Sorry for the confusion. Actually the behavior here is correct, i.e., if groupVersion="users/", we want to pass "" to api.Scheme.KnownTypes(), because we want to generate deep copies for internal object, which is registered using AddKnownTypes("", ...)